databuilder/tables/beta/tpp.py

```diff
@@ -58,6 +59,21 @@ class practice_registrations(EventFrame):
         ],
     )
 
+    def active_for_patient_at(self, date):
+        """
+        Return each patient's "active" practice registration as of the supplied date
+        """
+        spanning_regs = self.take(self.start_date <= date).drop(self.end_date <= date)
```

Comment:
I think this is identical to the logic that QOF uses to define registrations (this is good). The only difference is that QOF specifies another thing (GMS) which I don't entirely understand. 

> Select patients who meet either of the criteria below:
> - Registered for GMS prior to or on the achievement date and did not subsequently deregister from GMS (currently registered for GMS).
> - Registered for GMS prior to or on the achievement date and subsequently deregistered from GMS after the achievement date (previously registered for GMS).
(i.e. patients who were registered for GMS on the achievement date)
>
> Reject the remaining patients.

But I confused myself by thinking the smaller than in `.drop(self.end_date <= date)` is not doing what we want so I'm posting it and am moving on with my review.

```suggestion
        spanning_regs = self.take(self.start_date <= date).drop(self.end_date < date)
```